src/store/index.js

```diff
@@ -5,19 +5,34 @@ import { INITIALIZE_PNP } from './action-types.js'
 import { UPDATE_AVAILABLE } from './mutation-types'
 
 Vue.use(Vuex)
+let edgeDeviceVersion = require('@/../package.json').version
+
+try {
+  fetch('http://localhost:8778/api/status')
```

Comment:
@vickywane this is still a problem, as I mentioned in another comment. It is also what causes the Lighthouse fail.

Your are using the regular HTML 5 fetch() API instead of the p2p fetch over webrtc. Look closer into the mechanics of http access over webrtc.

![Screen Shot 2021-03-12 at 2 51 32 PM](https://user-images.githubusercontent.com/2234901/110997228-b7d1eb00-8342-11eb-8254-b1c9e12ccba1.png)

Reply:
![Screen Shot 2021-03-12 at 2 57 09 PM](https://user-images.githubusercontent.com/2234901/110997624-48a8c680-8343-11eb-8379-352db43165ad.png)